homeassistant/components/nissan_leaf/switch.py

```diff
@@ -0,0 +1,99 @@
+"""Charge and Climate Control Support for the Nissan Leaf."""
+import logging
+
+from homeassistant.components.nissan_leaf import (
+    DATA_CHARGING, DATA_CLIMATE, DATA_LEAF, LeafEntity)
+from homeassistant.helpers.entity import ToggleEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['nissan_leaf']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Nissan Leaf switch platform setup."""
+    _LOGGER.debug(
```

Comment:
Remove this.

Reply:
Fixed with commit https://github.com/filcole/home-assistant/commit/119b2a3b20db6ea4cf05fda29921930c5e27bc86